### DISCUSSION TITLE
Didn't make the cut for the Google Season of Docs

### DISCUSSION THREAD

**Discussion by EricZinda** [delph-in/docs#118] [Docs Working Group]
Google decided not to give us a grant for their Season of Docs effort for the [proposal we submitted](https://delph-in.github.io/docs/garage/GoogleSeasonOfDocs2023ProjectProposal/). When I asked if they could give us any feedback that might help us be accepted next go around they said "Read the Season of Docs Docs" which was kind of appropriate, I guess.

Good news is we have a writeup of what we'd like in a set of "getting started tutorials" that we can use moving forward, though.

**Comment by olzama** [comment]
It was a good and worthy attempt! Thank you, Eric!

**Reply by EricZinda** [reply]
Thanks Olga!
